doc/i2p.md

```diff
@@ -0,0 +1,71 @@
+# I2P support in Bitcoin Core
+
+It is possible to use Bitcoin Core to accept connections from and make
+connections to other Bitcoin nodes through the
+[I2P network](https://en.wikipedia.org/wiki/I2P).
+
+[I2P glossary](https://geti2p.net/en/about/glossary) may be useful to get
+started with I2P terminology.
+
+## Run Bitcoin Core with an I2P router (proxy)
+
+A running I2P router (proxy) with [SAM](https://geti2p.net/en/docs/api/samv3)
+enabled is required (there is an [official one](https://geti2p.net) and
```

Comment:
This official link has one 'download' button, it will open https://geti2p.net/en/download which has options for different OS. If you try installing `i2p` for Ubuntu from here it won't work. Others also look like .jar files